docs/usage/configuration-options.md

```diff
@@ -119,6 +119,12 @@ See [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-s
 
 If configured, Renovate will take a random sample of given size from assignees and assign them only, instead of assigning the entire list of `assignees` you have configured.
 
+## autoReplaceRegexGlobalMatch
```

Comment:
```suggestion
## autoReplaceGlobalMatch
```

A shorter name would be better, and "regex" is too much details for most users

Reply:
Done deal 🤝